jupyterlab_automl/src/components/model_properties.tsx

```diff
@@ -1,221 +1,35 @@
-import {
-  LinearProgress,
-  Table,
-  TableBody,
-  TableCell,
-  TableRow,
-  Collapse,
-  IconButton,
-  TableHead,
-  Box,
-} from '@material-ui/core';
-import { KeyboardArrowUp, KeyboardArrowDown } from '@material-ui/icons';
+import { LinearProgress } from '@material-ui/core';
 import * as React from 'react';
 import { Model, ModelService, Pipeline } from '../service/model';
+import { PipelineProperties } from './pipeline_properties';
 
 interface Props {
   model: Model;
   value: number;
   index: number;
+  pipeline: Pipeline;
 }
 
 interface State {
   hasLoaded: boolean;
   isLoading: boolean;
-  modelDetails: any[];
-  transformationOptions: any[];
+  pipeline: Pipeline;
   open: boolean;
 }
```

Comment:
I am a little confused why pipeline is a prop and also a state variable. We can probably get rid of both the `pipeline` and `open` state variables